backend/src/controllers/post.controller.js

```
@@ -0,0 +1,164 @@
+      }:base64,${imageFile.buffer.toString("base64")}`;
+
+      const uploadResponse = await cloudinary.uploader.upload(base64Image, {
+        folder: "social_media_posts",
+        resource_type: "image",
+        transformation: [
+          { width: 800, height: 600, crop: "limit" },
+          { quality: "auto" },
+          { format: "auto" },
+        ],
+      });
+      imageUrl = uploadResponse.secure_url;
```

Biome (1.9.4)

[error] Can't assign imageUrl because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)